masterlist.yaml

```
@@ -6572,3 +6572,24 @@ plugins:
     clean:
       - crc: 0x17ED1071
         util: '[FO4Edit v4.0.4](https://www.nexusmods.com/fallout4/mods/2737)'
+
+  - name: 'VtawWardrobe5.esp'
+    url:
+      - link: 'https://www.nexusmods.com/fallout4/mods/43844/'
+        name: 'Vtaw Wardrobe 5 - CBBE - BodySlide (Nexus Mods)'
```

There aren't multiple locations with the same name but different links, so you don't need to specify the website:

```suggestion
        name: 'Vtaw Wardrobe 5 - CBBE - BodySlide'
```